nbs/02_match_class.ipynb

```diff
@@ -21,7 +21,16 @@
    "cell_type": "code",
```

Comment:
<p>this example is confusing, since letters are normally used for variables and numbers are usually used for IDs. Please change the example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>